packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -364,9 +314,11 @@ export default class ComponentsTableRoute extends Route {
   </h4>
   <Hds::Table>
     <:head>
-      <th>Cell Header</th>
-      <th>Cell Header</th>
-      <th>Cell Header</th>
+      <tr>
+        <th>Cell Header</th>
+        <th>Cell Header</th>
+        <th>Cell Header</th>
+      </tr>
```

Comment:
Maybe this example should use `<Hds::Table::Row>` and `<Hds::Table::Th>` too?

Reply:
I would document it in some "subtle" way, but not suggest it (at lest for now). 

What do you think @alex-ju ?

Reply:
If I understand correctly, we have two things that we can potentially highlight (either in the 'Showcase' or in the 'How to use' section). One is that they can use the component with or without a `@model` passed to it and the other one is that they can use a subcomponent or simple HTML elements to define the rows and cells. What Melanie did here was to couple the two in a more 'rigid'/structured example and a more 'flexible'/custom one.

If we want to support simple HTML elements (`tr`, `th`) I think it's valuable to 'showcase' them so we can keep an eye on potential visual regression, if we're not committed to supporting them yet then maybe there's not much value. Having them in the showcase doesn't mean we promote them, it's more that we make sure they work as expected.